dlrover/python/common/grpc.py

```diff
@@ -508,6 +508,15 @@ class ElasticRunConfig(Message):
     configs: Dict[str, str] = field(default_factory=dict)
 
 
+@dataclass
+class InfoEvent(Message):
```

Comment:
Should change a name. Not all these event are 'INFO' type as there is also a ```event_type``` field definition here.